Selenium/Task50/Task50-5/TestTask50-5.cs

```diff
@@ -0,0 +1,45 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using System;
+using System.Threading;
+
+namespace Task50_5
+{
+    [TestFixture]
+    public class TestTask50_5
+    {
+        private string option1 = "California";
+        private string option4 = "New York";
+        private string option5 = "Ohio";
+
+        IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            driver = new ChromeDriver();
+            this.driver.Manage().Window.Maximize();
+        }
+
+        [Test]
+        [Description("Verifies that 3 random options are selected")]
+        public void Verify3RandomOptionsSelected()
+        {
+            Task50_5Page tp = new Task50_5Page(driver);
+            tp.NavigateToTask50_5Page();
+            tp.Select3Options(option1, option4, option5);
+
+            Assert.IsTrue(tp.option1.Selected);
```

Comment:
Just call AllSelectOptions property at your Select Web Element and you will get all of them. After that you may validate that returned number is 3 by using Assert method

Reply:
Fixed